src/survey.py

```diff
@@ -172,3 +172,33 @@ async def add_receive_role_if_exists(client, member, survey_id):
             f"Adding role ({survey_receive_role.name}) to user {member_to_string(member)}"
         )
         await member.add_roles(survey_receive_role)
+
+
+# Create channel which can be seen only by this member, bot and admins
+async def create_channel_for_survey(guild, member, channel_name):
```

Comment:
This could be reused in `welcome.py` instead of `_create_welcome_channel` which could then be deleted.